### PR TITLE
Create vars.md

### PR DESCRIPTION
**Pull request by SayalGorania**
Created vars.md file and added content

### Description

Added vars.md file to describe the built-in vars function in Python. Closes issue #2389 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+**vars()** is a built-in python function that returns the __dict__ attribute of an object. The __dict__ attribute is a dictionary containing the object's changeable attributes.
+
+## Syntax
+```pseudo
+vars(object)
+```
+
+## Example
+
+Use vars() to return the __dict__ attributes of the object 'Person'
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:22]:
```suggestion
The following example uses `vars()` to return the `__dict__` attributes of the object `Person`:
```

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```codebyte/python
+	class Person:
+		name = "John"
+		age = 36
+		country = "norway"
+
+	x = vars(Person)
+	
+	>>>{'__module__': '__main__', 'name': 'John', 'age': 36, 'country': 'norway', '__dict__': <attribute '__dict__' of 'Person' objects>, '__weakref__': <attribute '__weakref__' of 'Person' objects>, '__doc__': None}
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:32]:
```suggestion
```
That is not actual code or part of the codebyte. Remove it. Also add a line to output the value of `x` to the user.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@SayalGorania I did a first review and left a few comments.

**Comment by SayalGorania** [comment]
@SSwiniarski  I have reviewed an tried to implement your comments and suggestions.

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```pseudo
+vars(object)
+```
+
+Here, 'object' refers to an instance of a particular class.
+
+## Example
```
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:25]:
```suggestion
## Codebyte Example
```

**Review by SSwiniarski — APPROVED** [review]
@SayalGorania looks good for a second review.

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'vars()'
+Description: 'Returns the __dict__ attirbute of an object.'
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:3]:
```suggestion
Description: 'Returns the __dict__ attribute of an object.'
```

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Subjects: 
+  - 'Computer Science'
+Tags: 
+  - 'Functions'
+  - 'Attributes'
+  - 'Objects'
+CatalogContent: 
+  - 'learn-python'
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:11]:
```suggestion
  - 'learn-python-3'
```

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  - 'paths/computer-science'
+---
+
+**`vars()`** is a built-in python function that returns the `__dict__` attribute of an object. The `__dict__` attribute is a dictionary containing the object's changeable attributes.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:15]:
```suggestion
**`vars()`** is a built-in Python function that returns the `__dict__` attribute of an object. The `__dict__` attribute is a dictionary containing the object's mutable attributes.
```

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```pseudo
+vars(object)
+```
+
+Here, 'object' refers to an instance of a particular class.
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:23]:
```suggestion
Here, `object` refers to an instance of a particular class.
```

File: content/python/concepts/built-in-functions/terms/vars/vars.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example
+
+The following example uses `vars()` to return the `__dict__` attributes of the object `Person`:
+
+```codebyte/python
+	class Person:
```
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/vars/vars.md:30]: the code should be shifted to the left

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SayalGorania I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by SayalGorania** [comment]
@caupolicandiaz I have reviewed your comments and tried to make the relevant changes.

**Review by CBID2 — APPROVED** [review]
Looks good to me! 😊

**Review by caupolicandiaz — APPROVED** [review]
@SayalGorania Looks good.

**Comment by SSwiniarski** [comment]
@SayalGorania Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/python/built-in-functions/vars

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**